packages/@uppy/dashboard/src/components/Slide.js

```diff
@@ -43,14 +45,14 @@ class Slide extends Component {
 
       this.animationFrame = requestAnimationFrame(() => {
         // Force it to render before we add the active class
-        this.base.getBoundingClientRect()
+        // this.base.getBoundingClientRect()
 
-        this.setState({
+        this.setState(() => ({
           className: `${transitionName}-enter ${transitionName}-enter-active`,
-        })
+        }))
 
         this.enterTimeout = setTimeout(() => {
-          this.setState({ className: '' })
+          this.setState(() => ({ className: '' }))
```

Comment:
I don't think it's necessary to use a callback here:
- preact don't use them in their documentation examples: https://preactjs.com/guide/v10/components/#class-components
- React specifies it's useful only if the state mutation depends on the current/previous state: https://reactjs.org/docs/react-component.html#setstate